From 6666cff61d64261e9ca64faad22dfd30ab097cb0 Mon Sep 17 00:00:00 2001
From: Mete Alpaslan <mete@pardus.org.tr>
Date: Thu, 2 Dec 2010 16:50:31 +0200
Subject: [PATCH 6/6] yali.flags:Add enabling default installation

If yali=default is set in cmdline, installation run as STEP_DEFAULT.
---
 yali/flags.py |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/yali/flags.py b/yali/flags.py
index 28b307e..caaac75 100644
--- a/yali/flags.py
+++ b/yali/flags.py
@@ -36,6 +36,8 @@ class Flags:
             self.__dict__['flags']['install_type'] = 3
         elif options.has_key("rescue") :
             self.__dict__['flags']['install_type'] = 4
+        elif options.has_key("default") :
+            self.__dict__['flags']['install_type'] = 0
 
         if options.has_key("theme"):
             self.__dict__['flags']['theme'] = options["theme"]
-- 
1.7.2.2

